features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -222,28 +222,57 @@ against unverified tokens.
 
 ## Results
 
-Sign-in confirmation was deployed over several train cycles.
-The feature was turned on at 6-24-2016 for 2% of our users and
-gradually increased to 100%. During the deployment we monitored
-the sign-in success and error rates to ensure that users were
-not negatively impacted.
+Sign-in confirmation was deployed over several
+train cycles. The feature was turned on at 6-24-2016
+for 2% of our users and gradually increased to 100%.
+During the deployment we monitored the sign-in success
+and error rates to ensure that users were not negatively
+impacted.
 
 To help gauge the success of the feature, several dashboards
 were created.
 
 ### Sync sign-in success rate
 
-Pre sign-in confirmation metrics show that we had between a 42% to 52%
-sync sign-in success rate. From [first dashboard](https://app.datadoghq.com/dash/163668/fxa-content---signin-confirmation), with sign-in confirmation, we have maintained this sign-in rate.
+Pre sign-in confirmation metrics show that we had
+between a 42% to 52% sync sign-in success rate.
```

Comment:
> Why are there 3 different login success events?
> Why are there 2 different form submit events?

This is the most common question we have about the metrics and why I proposed https://github.com/mozilla/fxa-content-server/issues/4163. 

Most reported event names contain a built in screen name. This is so we know on which screen an event originates. Logging `success` without any additional context is a pointless metric - "what kind of success?" is the obvious question that follows. Way back when, there was only 1 way to sign in, from the `/signin` screen. At that time it was sufficient to have `fxa.content.signin.success`, we knew what `signin.success` meant and where it was coming from. Users can now sign in from 6 different endpoints: `/signin`, `/force_auth`, `/signup`, `/oauth/signin`, `/oauth/force_auth`, and `/oauth/signup`. When we started to add endpoints, we still wanted to know from which screen the user was successfully signing in, so we "extended" the event name - `fxa.content.<screen_name>.signin.success`. It has become unruly because there is no single `fxa.content.signin.success` anymore.